boto3utils/s3.py

```diff
@@ -116,36 +122,88 @@ def upload_json(self, data, url, extra={}, **kwargs):
         finally:
             rmtree(tmpdir)
 
-    def get_object(self, bucket, key):
+    def get_object(self, bucket, key, extra_args={}):
         """ Get an S3 object """
+        extra_args = deepcopy(extra_args or {})
+
         if self.requester_pays:
-            response = self.s3.get_object(Bucket=bucket,
-                                          Key=key,
-                                          RequestPayer='requester')
-        else:
-            response = self.s3.get_object(Bucket=bucket, Key=key)
-        return response
+            extra_args["RequestPayer"] = "requester"
 
-    def download(self, uri, path=''):
+        return self.s3.get_object(Bucket=bucket, Key=key, **extra_args)
+
+    def download(self, uri, path='', extra_args={}):
```

Comment:
I think a more pythonic way of doing this would be to accept `**kwargs` rather than `extra_args` as a dictionary, then pass **kwargs into boto3 `download_file` (after modifying with parsed queries).

Reason being here is that `download` as a function really is just a wrapper around `downlaod_file`, and since it accepts any keywords that accepts, the more standard approach would be to accept and pass **kwargs.

Reply:
yeah makes sense to me.